decidim-proposals/app/controllers/concerns/decidim/proposals/admin/filterable.rb

```diff
@@ -55,13 +53,21 @@ def filters_with_values
           # Cannot user `super` here, because it does not belong to a superclass
           # but to a concern.
           def dynamically_translated_filters
-            [:scope_id_eq, :category_id_eq, :valuator_role_ids_has]
+            [:scope_id_eq, :category_id_eq, :valuator_role_ids_has, :proposal_state_id_eq]
+          end
+
+          def proposal_state_ids
+            ProposalState.where(component: current_component).pluck(:id)
           end
 
           def valuator_role_ids
             current_participatory_space.user_roles(:valuator).pluck(:id)
           end
 
+          def translated_proposal_state_id_eq(state_id)
+            translated_attribute(ProposalState.find_by(id: state_id)&.title)
```

Comment:
```suggestion
            translated_attribute(ProposalState.find_by(component: current_component, id: state_id)&.title)
```

Reply:
Should we apply this auto suggestion @alecslupu ?

Reply:
Is ok, we can skip it.